ftl/common/ftl_util.py

```diff
@@ -27,3 +31,18 @@ def __enter__(self):
     def __exit__(self, unused_type, unused_value, unused_traceback):
         end = time.time()
         logging.info('%s took %d seconds', self.descriptor, end - self.start)
+
+
+def folder_to_layer_sha(pkg_dir, pkg_mngr_name):
+    tar_path = tempfile.mktemp()
+    with Timing("tar_%s_package" % pkg_mngr_name):
```

Comment:
It's a bit weird we need the package manager name here - there's nothing else that uses this except the logging statement. Can we remove that as a parameter?

Reply:
done